livekit-agents/livekit/agents/voice/room_io/_input.py

```diff
@@ -50,7 +58,7 @@ def on_attached(self) -> None:
             "input stream attached",
             extra={
                 "participant": self._participant_identity,
-                "source": rtc.TrackSource.Name(self._track_source),
```

Comment:
I think it's fine to keep the singular `source`. for consistency when using Datadog or other search tools, it's far easier to search for `source` always, instead of trying to figure out to specify sources or source